cadc-gms/build.gradle

```diff
@@ -0,0 +1,27 @@
+plugins {
+    id 'java'
+    id 'maven'
+    id 'maven-publish'
+    id 'com.jfrog.bintray' version '1.8.4'
+}
+
+repositories {
+    jcenter()
+    mavenLocal()
+}
+
+sourceCompatibility = 1.7
+
+group = 'org.opencadc'
+
+version = '1.0.0'
+
+dependencies {
+    compile 'log4j:log4j:1.2.+'
+    compile 'org.opencadc:cadc-util:[1.1,)'
+    testCompile 'junit:junit:4.+'
+}
+
+configurations {
```

Comment:
There are either several more excludes of things cadc-util pulls in or none because end applications should be resposible for excluding... I assume this just makes the dependency list in the pom smaller which would be a good thing to do if we did it everywhere.